service/informer/informer.go

```diff
@@ -229,25 +237,25 @@ func containsEmptyItems(projects []string) bool {
 	return false
 }
 
-func ensureProject(projects []draughtsmantprspec.Project, project draughtsmantprspec.Project) []draughtsmantprspec.Project {
+func ensureProject(projects []draughtsmantprspec.Project, project draughtsmantprspec.Project) ([]draughtsmantprspec.Project, bool) {
```

Comment:
I would appreciate some godoc here - when is true/false returned?

Reply:
Added godoc.